grommet-leaflet/src/layers/Cluster.jsx

```diff
@@ -13,7 +13,7 @@ const StyledContainer = styled(Box)`
   }
 `;
 
-const StyledLabel = styled(Text)`
+const StyledLabel = styled(Paragraph)`
```

Comment:
Wondering about this if label is generally a short string or number? 

Or is this part of broader move to use `p` over `span`?

Reply:
This was a holdover for when Text was throwing a console warning due to a useLayoutEffect being used. This has been resolved in another PR. Changing to Text.